reconcile/test/test_version_bump.py

```diff
@@ -1,18 +0,0 @@
-import os
```

Comment:
why this file got deleted?

Reply:
The test was failing, and I thought instead of fixing it, I would remove it.

Reply:
Nevertheless, I won't die on that hill. [a96b11c](https://github.com/app-sre/qontract-reconcile/pull/5111/commits/a96b11c4558714879c7d64558fad95e4d7cdb798) fixes it

Reply:
I'll remove this test in another PR.